### PR TITLE
Remove legacy Zedwallet and promote Zedwallet++ out of beta

### PR DESCRIPTION
**Pull request by brandonlehmann**

### CODE REVIEW

File: src/serialization/SerializationTools.h [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #include <serialization/BinaryOutputStreamSerializer.h>
 #include <serialization/CryptoNoteSerialization.h>
 #include <vector>
-#include <zedwallet/Types.h>
+#include <ZedwalletTypes.h>
```
Comment (zpalmtree) [inline src/serialization/SerializationTools.h]: Can we not just remove the serialization stuff for zedwallet, or is it used elsewhere?

Comment (brandonlehmann) [reply src/serialization/SerializationTools.h]: I did, then the rabbit hole went deeper.

Comment (brandonlehmann) [reply src/serialization/SerializationTools.h]: I figure it can be totally removed once we do the same to turtle-service some day.

Comment (zpalmtree) [reply src/serialization/SerializationTools.h]: mehhhh

Comment (brandonlehmann) [reply src/serialization/SerializationTools.h]: If you’d like, I’ll get my helmet out.

Comment (zpalmtree) [reply src/serialization/SerializationTools.h]: Nah, I can handle it in a follow up PR, I probably know where it's used better

**Pull request merged**